style_editor.html

```diff
@@ -8839,6 +8923,8 @@
      if (bad_fps) bad_fps --;
   }
   if (bad_fps > 10 && use_graflex) {
+    popupMessage = "Struggling to render hilt model.<br>Switching to simpler design.<br>To re-enable Graflex model, go to Settings.";
```

Comment:
Does this work? (I don't see where the popupMessage variable is created.)
Also, there doesn't seem to be a need for a variable, just put the string in the showPopupMessage() call, right?

Reply:
oh yeah. that was silly.